web/public/js/services/dojo-utils.js

```diff
@@ -135,24 +135,56 @@ angular.module('cpZenPlatform').factory('dojoUtils', ['$location', '$translate',
     { id: 'search_engine',
       name: $translate.instant('Search Engine')},
     { id: 'volunteers',
-      name: $translate.instant('Other CoderDojo Volunteers')},
-    { id: 'organisations',
-      name: $translate.instant('Other Coding Organisations')},
+      name: $translate.instant('CoderDojo Volunteers')},
+    { id: 'foundation',
+      name: $translate.instant('Raspberry Pi Foundation / CoderDojo Foundation outreach')},
+    { id: 'social',
+      name: $translate.instant('Social media')},
     { id: 'future_learn',
       name: 'Future Learn'},
-    { id: 'developpers',
-      name: $translate.instant('Development Community')},
+    { id: 'employer',
+      name: $translate.instant('My employer (CSR scheme at work)')},
     { id: 'events',
-      name: $translate.instant('Events')},
+      name: $translate.instant('Event / Conference / Training')},
+    { id: 'library',
+      name: $translate.instant('Library or youth group')},
     { id: 'word_of_mouth',
       name: $translate.instant('Word of Mouth')},
+    { id: 'other',
+      name: $translate.instant('Other (comment field)')}
+  ];
+
+  dojoUtils.startingDojoSrcsWithDeprecated = [
```

Comment:
Wouldn't it be easier to extend this list with startingDojoSrcs rather than duplicating the definition ?

Reply:
🤦‍♂️